diktat-ruleset/src/test/kotlin/org/cqfn/diktat/ruleset/smoke/DiktatSaveSmokeTest.kt

```diff
@@ -82,7 +82,8 @@ class DiktatSaveSmokeTest : DiktatSmokeTestBase() {
                 }
 
                 val saveProcess = processBuilder.start()
-                val saveProcessResult = saveProcess.waitFor(TEST_TIMEOUT_SECONDS, TimeUnit.SECONDS)
+                // set some big timeout to wait more that JUnit can be configured to wait
+                val saveProcessResult = saveProcess.waitFor(1, TimeUnit.HOURS)
```

Comment:
So what's the point of this, anyway?

Once you've reached the time-out, everything below is dead code.

Reply:
currently test is failed on Windows is due to timeout in 10 seconds, but we have setup it to 25 seconds.
Since we have timeout for JUnit test -- it should be marked as failed tests by timeout.
Now I set timeout to some big value to be more that junit timeout -- so we get timeout only by junit engine

Reply:
didn't reproduced